0023-backend-v3.md

```diff
@@ -0,0 +1,239 @@
+
+A few years later, the `Sampler` and `Estimator` **primitives** were introduced. Instead of an abstraction for a physical object (the quantum computer) or a part of it (the QPU), the primitives introduced a higher level of abstraction for the two most common **tasks** or **access patterns** executed in quantum computers, these are: sampling from a state (defined as a circuit) and computing expectation values from observables on states. These were developed as standalone entities, independent of the `Backend` model, but effectively replaced the original backend implementation of the execution step, the famous `backend.run`. 
+
+<img src="./0023-backend-v3/overview-b.png" height="300">
+
+The image above summarizes the current superposition of the backend and primitives models at the execution ("run") step. This "unresolved superposition" between the backend and the primitives model disrupts the design and development process around backends and primitives. What's allowed and not allowed is not clear. There are multiple backend providers and primitive implementations (`qiskit`, `qiskit-ibm-runtime`, `qiskit-aer`, external providers), and users struggle to figure out what primitive to use, what backend to use, how do they interact with each other... On top of this, current backend implementations (such as the `IBMBackend`) do not respect the guidelines of their original abstract interface. This is the motivation for introducing a new `BackendV3` that better reflects the status quo but respects the essence and usability of the original backend proposal.
+
+Before diving into the specifics of the `BackendV2` design (current status) and its interaction with the primitives model, it's important to note that the `Backend` interface was designed with two broader goals:
+
+1. Providing a interface general enough to support any vendor (**vendor-agnostic**)
+2. Setting a level of abstraction that allows the **physical hardware to evolve** without affecting the abstract object
+
+Both these goals have been fulfilled, with a growing list of 3rd-party providers that consume the `Backend` interface, which has seamlessly supported hardware changes in basis gates, couplers, noise definitions, introduction of qubit resets, and more. The goal for `BackendV3` is to maintain these design principles.
+
+### What will be the outcome?
+
+A new `BackendV3` class. This class should be almost 100% backwards-compatible with the current transpilation stack (for which it must contain a `Target`), but not require a `.run` method, and instead provide primitive-friendly alternatives.
+
+Instead of a required `backend.run()`, the proposal for `BackendV3` implementations is to offer **primitive factory methods**: `backend.sampler()` and `backend.estimator()` that return their respective instantiated primitive classes (details TBD), as well as an optional `backend.run()` for 3rd party and local simulation backends (up to discussion).
```

Comment:
I think this is very important aspect, the difference between the migration from `BaseBackend` -> `BackendV1` and `BackendV1` -> `BackendV2` is that moving to `BackendV1` was a simple migration it changed very little. If there was only one lesson from `BackendV2` is that changing too much at once makes it exponentially harder to get everyone to migrate. So keeping this small and not breaking existing implementations so the migration is trivial this is a better way to evolve the interface.

Reply:
Added in [0427d3d](https://github.com/Qiskit/RFCs/pull/77/commits/0427d3da30df0bf05fcf405b92120e4cbfa4edc5)